processing/src/main/java/io/druid/query/Queries.java

```diff
@@ -33,6 +34,7 @@
 
 /**
  */
+@PublicApi
 public class Queries
```

Comment:
It's utility methods that are use in the implementations of most built-in queries and post-aggregators. If it's not a public api, then people that implement their own extension based on the built-in code (which is probably common) will accidentally use internal apis.